### DISCUSSION TITLE
新手求问关于120hz 4k显示器开启gsync的情况下mpv全屏ui帧率会降低到40

### DISCUSSION THREAD

**Discussion by nian555** [hooke007/mpv_PlayKit#327] [mpv 通用]
这种体验下鼠标很明显卡顿，不知道如何解决。用着就很难受。我已知关闭gsync可以解决此问题 但是平时玩游戏还要用到gsync，想请教如何解决。求请教下解决方法，在此感谢（另外还有一个小问题说一下 就是需要sdr模式下播放hdr视频自动转成hdr模式 播放sdr就正常播放，我目前的情况是播放视频sdr过曝，播放hdr需要手动开启系统hdr，很不方便 ）

以下是我的mpv.conf的hdr设置 之所以用d3d11是因为我用vulkan的话虽会激活hdr但是画面颜色过曝 d3d11是sdr电影过曝 现在就是两难的境地
#hwdec=vulkan-copy
vo=gpu-next
gpu-context=d3d11
#gpu-context=winvk
target-colorspace-hint=yes
profile-cond=p["video-params/sig-peak"]>1 and p["video-params/gamma"]=="pq"
profile-restore=copy
target-trc=pq
target-peak=1000

**Comment by hooke007** [comment]
FAQ已经写了
单独配置gsync关闭啊

**Reply by hooke007** [reply]
hdr的问题我也写了，仔细阅读。
tldr: 有hdr直出和sdr混合的全自动需求，我的意见是（如果vulkan无法正常使用，那就）别用mpv

**Reply by hooke007** [reply]
BTW 你的配置写法也不对，profile没有头

**Reply by nian555** [reply]
谢谢楼主 gsync已经解决 我没有仔细看问答 后面我有问题会先仔细看问答在做打算 hdr这个我再试试看 还有一个就是怎么可以播放iso格式问文件呢

**Reply by hooke007** [reply]
iso本质就是压缩包，能直接播放就是没问题，不能放就是不支持部分内部格式

**Comment by nian555** [comment]
> BTW 你的配置写法也不对，profile没有头

那应该怎么写呢 我也是网上查到的写法 就这么写了

**Reply by hooke007** [reply]
我仓库和我的包都有现成的案例不看，为什么要查网上的写法呢

我的建议是从头学起 https://hooke007.github.io/unofficial/mpv_profiles.html

**Reply by nian555** [reply]
嗯嗯 我仔细看看 就是白天才接触到这个强大的播放器 转来转去看到你发布的这个更好的mpv
